.pre-commit-config.yaml

```diff
@@ -0,0 +1,54 @@
+ci:
+  autofix_prs: false
+  autoupdate_schedule: 'monthly'
```

Comment:
The [docs](https://pre-commit.ci/#configuration-autoupdate_schedule) for this option seem to say that there's no way to turn this off. However, https://pre-commit.ci/lite.html seems to say that this `ci:` block can go away completely? I'm not sure if those docs are correct, it doesn't really make sense to me that the `lite-action` requires the full action with `uses: pre-commit/action@v3.0.0`.

Could you see if there's a way, and if not then set this to `quarterly` as the slowest document frequency?

Reply:
It can be removed!